cypress/integration/70-pipeline/ExecutionHistory.spec.ts

```diff
@@ -24,6 +24,8 @@ describe('Pipeline Execution History', () => {
   })
 
   it('loads a pipeline with no executions', () => {
+    cy.wait(3000) // page load sometimes takes time
```

Comment:
```suggestion
    cy.wait(3000) // page load sometimes takes time
```

Can you check once using `cy.visitPageAssertion()`, this is asserting the page element before making API calls.